### DISCUSSION TITLE
(Idle) after i switch tab

### DISCUSSION THREAD

**Discussion by ypgrimm** [Selbi182/SpotifyBigPicture#33] [Q&A] [answered]
I know it's not important whatsoever but I was wondering if there was a setting that would stop this and keep the song name so I can see what song is playing when doing something else just by looking at the tab name, just for the recordI do have idle mode off.

**Comment by Selbi182** [comment, marked as answer]
This was introduced to address a critical bug. When you had the tab hidden for a long time but still listened to a bunch of songs, the moment you would open the tab again, the entire browser would lag. Basically a massive memory leak.

**Reply by ypgrimm** [reply]
Ah I see okay thanks!
